src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiryConfirmation.tsx

```diff
@@ -13,7 +13,7 @@ export const ConsignmentInquiryConfirmation: React.FC = () => {
       <Spacer y={4} />
 
       <Button
-        width={["100%", "33%"]}
+        width={["100%", "20%"]}
```

Comment:
Is this a normal practice in force?

Reply:
I don't understand. Is there a better practice? @dariakoko

Reply:
@dariakoko I am not sure of a better practice. I've looked through the codebase. Do you know of a better approach?

Reply:
I would just expect anything that is not width 100% to be set up in palette. But now i think about it and it might be a bit of an overkill
I'm ok with your implementation

Reply:
Let's merge